pkg/cli/image/info/info.go

```diff
@@ -87,35 +91,67 @@ type InfoOptions struct {
 	SecurityOptions imagemanifest.SecurityOptions
 	FilterOptions   imagemanifest.FilterOptions
 
-	Images []string
-
-	FileDir string
+	operatorClient operatorv1alpha1client.OperatorV1alpha1Interface
 
-	Output string
+	Images   []string
+	FileDir  string
+	Output   string
+	ICSPFile string
 }
 
-func (o *InfoOptions) Complete(cmd *cobra.Command, args []string) error {
+func (o *InfoOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, args []string) error {
 	if len(args) < 1 {
 		return fmt.Errorf("info expects at least one argument, an image pull spec")
 	}
 	o.Images = args
-	return nil
-}
 
-func (o *InfoOptions) Validate() error {
-	return o.FilterOptions.Validate()
+	clientConfig, err := f.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+	o.operatorClient, err = operatorv1alpha1client.NewForConfig(clientConfig)
```

Comment:
This has to be lazy - we don't want to fail an `oc image` command because we couldn't create a client.  So you have to pass a `ToRESTConfig` method into your strategy.

Reply:
Actually, if you're not defaulting to the cluster strategy, you don't need this anymore, so you can remove.